packages/components/app/styles/components/tooltip.scss

```diff
@@ -0,0 +1,188 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TOOLTIP
+//
+
+:root {
+  // Dimensions:
+
+  // $tooltip-width: 192px;
+  --token-tooltip-max-width: 192px;
+
+  // $arrow-border-size: 7.5px;
+  --token-tooltip-arrow-border-size: 7.5px;
+
+  // $arrow-width: calc(#{$arrow-border-size} * 2); // 15 pixels
+  --token-tooltip-arrow-width: calc(var(--token-tooltip-arrow-border-size) * 2); // 15 pixels
+
+  // $arrow-height: calc(#{$arrow-border-size} * 1.07); // 8 pixels
+  --token-tooltip-arrow-height: calc(var(--token-tooltip-arrow-border-size) * 1.07); // 8 pixels
+
+  // $arrow-y-axis-distance: calc(0px - #{$arrow-border-size});
+  --token-tooltip-arrowy-axis-distance: calc(0px - var(--token-tooltip-arrow-border-size));
+
+  // Colors:
+
+  // $tooltip-background-color: var(--token-color-palette-neutral-700);
+  --token-tooltip-surface-primary: var(--token-color-palette-neutral-700);
+}
```

Comment:
Most of the styles are from Cloud-ui. I started add CSS custom properties to potentially replace the sass variables they had but not sure how useful it would be to expose all these different values so would like to discuss.

Reply:
Update: I changed the arrow style to use an included rounded version from tippy.js to match our design mocks. Most of the related styles for the previous arrow style are no longer needed with this change. I commented out these no longer relevant styles for now. If the updated rounded arrow style is approved I will go ahead and delete the old commented out styles.